### PR TITLE
add package py-lazy-object-proxy

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-lazy-object-proxy/package.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# This is a template package file for Spack.  We've put "FIXME"
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/py-lazy-object-proxy/package.py]: Can you remove the template placeholder text here?

Comment (healther) [reply var/spack/repos/builtin/packages/py-lazy-object-proxy/package.py]: done, sorry I missed that one

**Comment by healther** [comment]
reopen for travis to restart

**Comment by healther** [comment]
Travis says:
```The command "if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew ls --versions gcc    > /dev/null || brew install gcc;    fi" failed and exited with 1 during .```

@junghans this PR is required for #6493 to be useful, right now spack should complain about a missing package. Any idea how we can fix Travis?

**Pull request merged**